src/main/java/svc/data/citations/datasources/rejis/RejisCitationFilter.java

```diff
@@ -0,0 +1,29 @@
+package svc.data.citations.datasources.rejis;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Component;
+
+import svc.data.citations.datasources.rejis.models.RejisPartialCitation;
+
+@Component
+public class RejisCitationFilter {
+
+	public List<RejisPartialCitation> Filter(List<RejisPartialCitation> partialCitations){
+		return partialCitations.stream()
+				 .filter(c -> statusFilter(c))
+				 .collect(Collectors.toList());
+	}
+
+	private boolean statusFilter(RejisPartialCitation partialCitation){
+		boolean keepCitation = false;
```

Comment:
Normally I wouldn't care much but this method is so stinking simple to leave as 6 lines.

Can you just make it:

`return !partialCitation.caseStatus.equals("C")`